### DISCUSSION TITLE
23.12.14 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#89] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 유즈케이스 작성
- 내일은 어떤 일을 하나요?
  - 발표 준비
  - 노래 조회 기능 구현

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 리드미 작성을 위한 레퍼런스 조사
- 내일은 어떤 일을 하나요?
    - 발표 준비

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 구현한 것들 정리
* 내일은 어떤 일을 하나요?
  * 발표 준비
